deploy-dapp/deploy-to-azure.md

```diff
@@ -0,0 +1,58 @@
+# Deploying a Full Stack Dapp to Microsoft Azure
+
+In the previous [tutorials](https://kauri.io/collection/5b8e401ee727370001c942e3) we saw how to develop a full stack blockchain dapp. Now we deploy a dapp using [Microsoft Workbench Blockchain](https://azure.microsoft.com/en-us/features/blockchain-workbench/). This article outlines the steps to deploy a blockchain instance and deploy a blockchain dapp.
+
+## Deploy Blockchain Workbench
+
+Blockchain Workbench allows you to deploy a blockchain ledger along with a set of Azure services used to build a blockchain dapp. Once deployed, Blockchain Workbench provides access to client apps to create and manage users and blockchain applications.
+
+The following steps highlight on how to deploy blockchain workbench:
+1\.  Sign in to the  [Azure portal](https://portal.azure.com/). If you don't have an account, create one and select a suitable service/subscription plan. You can try the "free plan" for 30 days to test the waters.
+2\.  In the left pane, select _Create a resource_. Search for "Azure Blockchain Workbench"  in the _Search the Marketplace_ search bar. Select _Azure Blockchain Workbench_ and click _Create_.
+3\. Complete the basic settings and click _OK_.
+5\.  In _Advanced Settings_, choose if you want to create a new blockchain network or use an existing blockchain network and click _OK_  to finish Advanced Settings.
```

Comment:
@srisankethu This needs some further explanation. What's the difference between the options, what are the sub options?

Reply:
Addressed the issue